src/dropdown/index.js

```diff
@@ -175,6 +183,12 @@ class DropdownClab {
       : null;
   }
 
+  _compIsValidSelection(selected, options, label) {
+    return !isNil(selected) && Array.isArray(options)
+      ? options.filter(o => o[label] === selected[label]).length > 0
```

Comment:
@gabrieleborelli if the validation logic needs at least 1 option selected you could use the `.find` method which exits after the first match (if any):

```js
typeof options.find(o => o[label] === selected[label]) !== 'undefined'
```

Reply:
ok